scripts/globals/treasure.lua

```diff
@@ -255,7 +255,7 @@ xi.treasure.treasureInfo =
                     {
                         test = function(player)
                             return player:getQuestStatus(xi.questLog.JEUNO, xi.quest.id.jeuno.SCATTERED_INTO_SHADOW) == xi.questStatus.QUEST_ACCEPTED and
-                                player:getCharVar('scatIntoShadowCS') == 1 and not player:hasItem(xi.item.BEAST_COLLAR)
+                                player:getCharVar(Quest.getVarPrefix(3, 61) .. 'Prog') == 4 and not player:hasItem(xi.item.BEAST_COLLAR)
```

Comment:
Interesting - I had not seen this used except for rycharde in a discussion and in the commands folder... can we enum that? `player:getCharVar(Quest.getVarPrefix(JEUNO, SCATTERED_INTO_SHADOW ) .. 'Prog') == 4`

Reply:
https://github.com/LandSandBoat/server/blob/base/scripts/globals/treasure.lua#L526